doc/user_guide/io.rst

```diff
@@ -365,4 +367,43 @@ Therefore strongly reccommend to load using the ``.emi`` file instead.
 When reading an ``.emi`` file if there are several ``.ser`` files associated
 with it, all of them will be read and returned as a list.
 
+.. _blockfile-format:
+
+Blockfile
+---------
+
+HyperSpy can read and write the blockfile format from NanoMegas ASTAR software.
+It is used to store a series of diffraction patterns from scanning precession
+electron difraction (SPED) measurements, with a limited set of metadata. The
+header of the blockfile contains information about centering and distortions
+of the diffraction patterns, but is not applied to the signal during reading.
+
+.. warning::
```

Comment:
I would also warn the user about:

* This format only supporting uint8
* We provide no guarantee that they'll be able to open the file with the ASTRA software.